### PR TITLE
gitignore Web-SDK

### PR DESCRIPTION
**Pull request by Kirkkt**
Web-SDK doesn't work locally. To make it so we need to ln -s ... which will make the github version dirty.

gitignore is the way to do.

also, re-make to sync to the HEAD of web-sdk

@jakecadams

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,33 +1,35 @@
...
-node_modules
 .lock-wscript
-.DS_Store
+.map
+.project
+.settings
+Resources
+Thumbs.db
+Web-SDK
+Web-SDK/
```
Comment (jakecadams) [inline .gitignore]: I'm not a gitignore expert but can't we just do  Web-SDK/*

Comment (Kirkkt) [inline .gitignore]: sure thing.

In different systems symlink are treated differently, for example, github treats this Web-SDK as a solid file, and thus Web-SDK (Web-SDK/ doesn't work); but on linux, mac included, ln links are treated as folders, and thus Web-SDK/ (Web-SDK doesn't work).

**Comment by Kirkkt** [comment]
ping @jakecadams

File: .gitignore (hunk 1 above)
Comment (jakecadams) [inline .gitignore]: cool good to know!

**Comment by jakecadams** [comment]
:+1:

**Comment by Kirkkt** [comment]
close because I decided to push directly into master

**Pull request closed without merging**